docs/database/server/server-v3.sql

```diff
@@ -660,6 +660,9 @@ ON [Post] ([channel_id])
 CREATE INDEX [idx_Post_pending_post_id]

```

Comment:
@larkox @enahum should we make the files

server-v6.pdf
server-v6.rtf
server-v6.sql

(0/5) on this.  It hasn't been done since v3 obviously.

Or should we just stop worrying about different versions and just make the file server.* without any -v3.  This way, we can always go back via git blame to see the historical changes.

Reply:
> @larkox @enahum should we make the files
> 
> server-v6.pdf server-v6.rtf server-v6.sql
> 
> (0/5) on this. It hasn't been done since v3 obviously.
> 
> Or should we just stop worrying about different versions and just make the file server.* without any -v3. This way, we can always go back via git blame to see the historical changes.

Nah, I think is good for now.. we can revisit the docs at any moment in the future.